packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -0,0 +1,103 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny', 'large');
+const ignoreContrast = {
+  'color-contrast': { enabled: false },
+};
+
+const shouldTheme = (customProperties) => {
+  Object.entries(customProperties).forEach(([key, value]) => {
+    it(`themed [${key}]`, () => {
+      browser.setCSSCustomProps({ [key]: value });
+      expect(browser.checkElement('[data-reactroot]')).to.matchReference();
+    });
+  });
+};
+
+describe('Alert', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/tests/alert-tests/default'));
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports, rules: ignoreContrast });
```

Comment:
Do we need to disable ignoreContract on each test?

Reply:
I included it on each invalid test

Reply:
I added an issue for this and put it next to the rule so we know when to remove: https://github.com/cerner/terra-core/issues/1057